mantidproject/mantid
# ISIS Reflectometry GUI: Added buttons to expand/collapse all groups

This PR adds two buttons to the interface on the `Runs` tab:
- Expand groups - expands all currently closed groups in the table.
- Close groups - closes all currently open groups in the table.

**To test:**
- Code review.
- The interface can be accessed from Interfaces->Reflectometry->ISIS Reflectometry.
- Add several groups to the table. Test that each of the new buttons opens and closes all groups respectively.
- Check that documentation has been updated accordingly.

Fixes #18825.

*Release notes have been updated.*

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
